fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -347,6 +347,7 @@ terminal String
     KW_DISTRIBUTION,
     KW_DIV,
     KW_DO,
+    KW_DORIS_INTERNAL_TABLE_ID,
```

Comment:
should add into non-reserved keyword list, and add same keyword for Nereids in `src/main/antlr4/org/apache/doris/nereids/DorisParser.g4`